core/src/main/java/org/elasticsearch/cluster/action/shard/ShardStateAction.java

```diff
@@ -234,6 +234,11 @@ public void clusterStateProcessed(String source, ClusterState oldState, ClusterS
         }
 
         @Override
+        public String describeTasks(List<ShardRoutingEntry> tasks) {
+            return tasks.stream().map(entry -> entry.getShardRouting().toString()).reduce((s1, s2) -> s1 + ", " + s2).orElse("");
```

Comment:
we already log this as warn here - https://github.com/elastic/elasticsearch/pull/19255/files/4e270f69782fc6bdd9b29a564eb8b061d6ebe12b#diff-3a4f751f360271846c44772fb64294b1R184  , I felt it's better to be cocise.